src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -703,4 +703,14 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>Clearing NuGet plugins cache: {0}</value>
     <comment>{0}: Plugins cache path</comment>
   </data>
+  <data name="Error_RestoreInLockedMode" xml:space="preserve">
+    <value>Packages lock file is inconsistent with project dependencies so restore can't be run in locked mode. Please disable FreezeLockFileOnRestore MSBuild property or pass explicit --reevaluate flag to run restore to update the lock file.</value>
+  </data>
+  <data name="Error_PackageValidationFailed" xml:space="preserve">
+    <value>Package {0} sha512 hash validation failed. Package has been changed since the last restore.</value>
```

Comment:
sha changed because something changed in the package across different sources or single source or different machines... So i think from NuGet perspectives, it's appropriate to say that package changed from last restore.

Reply:
I don't agree. 
Let's discuss this offline. 

Changed could be interpreted as the package itself has been changed or tampered with and we don't know that.

Reply:
Discussed offline, agreed for `The package is different than the last restore.`